internal/work/errors.go

```diff
@@ -33,3 +33,15 @@ func newStringMapTypeConversionError(parameter interface{}) typeConversionError
 		t: reflect.TypeOf(map[string]string{}),
 	}
 }
+
+var ErrInvalidWorkerName WorkerError = "invalid worker name"
+
+type WorkerError string
```

Comment:
The name of error looks to generic, but it is related only to invalid name directive (not name of worker). The name of worker could be correct, but this error should be still raised.

I believe that code could be simplified. I propose something like this:

```go
type InvalidDirectiveNameError struct {
	InvalidName string
}

func (e InvalidDirectiveNameError) Error() string {
	return fmt.Sprintf("invalid directive name: %s", e.InvalidName)
}
```